src/anm/animation/animation.js

```diff
@@ -166,11 +167,34 @@ Animation.prototype.traverse = function(visitor, data) {
  *
  * @param {Function} visitor
  * @param {anm.Element} visitor.child
+ * @param {Boolean} visitor.return if `false` returned, stops the iteration. no-`return` or empty `return` both considered `true`.
  * @param {Object} [data]
  */
 Animation.prototype.each = function(visitor, data) {
     for (var i = 0, tlen = this.tree.length; i < tlen; i++) {
-        visitor(this.tree[i], data);
+        if (visitor(this.tree[i], data) === false) break;
```

Comment:
Tests are not the universal pill. Especially in JS. Sometimes you just need to trust yourself.

Again, it's a matter of decision, not a test. Returning `false` to skip element is quite common practice for JS, also it's properly noticed in docs. And, it's common to omit `return` in dynamic languages. You can't test users' code, who usually omit/forget `return` (like in 90% cases), especially while using `.each`.